src/controllers/public/movies.controllers.js

```diff
@@ -0,0 +1,33 @@
+import moviesJson from "../../json/movies-series/movies.json" assert { type: "json" };
+import { getPaginatedPayload } from "../../utils/helpers.js";
+import { ApiError } from "../../utils/ApiError.js";
+import { ApiResponse } from "../../utils/ApiResponse.js";
+import { asyncHandler } from "../../utils/asyncHandler.js";
+
+const getMovies = asyncHandler(async (req, res) => {
+  const page = +(req.query.page || 1);
+  const limit = +(req.query.limit || 10);
```

Comment:
Add more query parameters to make this api more dynamic. refer: https://github.com/hiteshchoudhary/apihub/blob/d0388a69e378e9eb5d1e5a4020fff2d65c0adf86/src/controllers/public/book.controllers.js#L7